pkg/cloud/cloud.go

```diff
@@ -1385,6 +1374,33 @@ func capIOPS(volumeType string, requestedCapacityGiB int64, requestedIops int64,
 	return iops, nil
 }
 
+func calculateGP3ReconciledThroughput(sizeGiB int64) int64 {
+	newThroughput := getGP3ReconciledThroughput(sizeGiB)
+	newThroughput = capThroughput(VolumeTypeGP3, sizeGiB, newThroughput, gp3MinTotalThroughput, gp3MaxTotalThroughput)
```

Comment:
capThroughPut seems unnecessary here, as the lower and upper boundaries are already enforced in the `getGP3ReconciledThroughput`?